docs/how_to/create_a_quantum_circuit.rst

```diff
@@ -0,0 +1,167 @@
+
+########################
+Create a quantum circuit
+########################
+
+This guide shows how to initialize a quantum circuit.
+
+There are two ways to create a :class:`~.QuantumCircuit` object:
+
+* Specifying the number of qubits and bits.
+* Creating :class:`~.QuantumRegister`\ s and :class:`~.ClassicalRegister`\ s.
+
+Create from number of qubits and bits
```

Comment:
```suggestion
Specifying the number of qubits and classical bits
```
I think the subsection title should match or at least use similar words as the two methods mentioned above.